packages/mongodb-chatbot-server/src/routes/conversations/addMessageToConversation.ts

```diff
@@ -44,6 +44,7 @@ export const DEFAULT_MAX_USER_MESSAGES_IN_CONVERSATION = 7; // magic number for
 export type AddMessageRequestBody = z.infer<typeof AddMessageRequestBody>;
 export const AddMessageRequestBody = z.object({
   message: z.string(),
+  clientContext: z.object({}).passthrough().optional(),
```

Comment:
tangential not for this PR: i think it'd be nice if for these 'custom' types, we had them be passable zod types, so you can specify the shape for your implementation, and then it's automatically validated on server